scripts/bootstrap-aio.yml

```diff
@@ -142,21 +142,27 @@
 
     - name: Add target user vars to user_variables_overrides
       set_fact:
-        user_variables_overrides: "{{ user_variables_overrides|combine(target_user_variables_overrides[TARGET])}}"
+        user_variables_overrides: "{{ user_variables_overrides|combine(target_user_variables_overrides[target])}}"
       when:
-        - TARGET is defined
-        - TARGET in target_user_variables_overrides
+        - target is defined
```

Comment:
So this was a whole thing where I was trying to get the playbook to take TARGET as either a variable you pass with vars or through the environment. Due to uncreative naming choices, I made the new value used target (since if I had `TARGET: "{{ TARGET | default(...`, that throws recursive errors). I'll change it to be more consistent and have clearer names.